integration_tests/pages/page.ts

```diff
@@ -511,4 +513,37 @@ export default abstract class Page {
       this.assertDefinition('Prison', person.prisonName)
     })
   }
+
+  shouldShowOutOfServiceBedDetails(bedDetails: OutOfServiceBed | OutOfServiceBedRevision) {
+    if (bedDetails.startDate) {
```

Comment:
Again I wonder if we should be a bit stricter with our terminology to avoid future confusion when reading this: i.e. this variable is more correctly an `outOfServiceBedDetails` isn't it?